backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColor.kt

```diff
@@ -22,80 +22,116 @@
 package net.skyscanner.backpack.compose.tokens
 
 import androidx.compose.ui.graphics.Color
+import kotlin.Deprecated
 
 public object BpkColor {
+  @Deprecated("This colour is now deprecated. Please switch to the new semantic colours - see internal New Colours documentation")
```

Comment:
I believe it's possible to use the compile-time constant here instead. Please consider it.

Reply:
i'll take a look